### ISSUE TITLE
[DOCS]: improve the contributing guide 

### ISSUE THREAD

**Issue by CBID2** [Learn-Build-Teach/deals-for-devs#209]
## Issue Description 
The instructions in the Contributing guide  needs to have prerequisites and extra steps. Also, the images were missing alt text and a couple of punctuation errors 
## Screenshots or Examples (if applicable)
N/A
## Proposed Solution (optional)
 If you have a proposed solution for the documentation issue, please provide it here. This can be helpful for speeding up the resolution process.
N/A
